docs/Q-and-A.md

```diff
@@ -292,6 +292,27 @@ We will be adding more graphical readouts of these things in the Explorer over t
 # **Q: Does an ION user node need bitcoin? or just IPFS?
 It needs bitcoin, but it can prune most of Bitcoin's data: all data before block 667000, and all non-ION transactions thereafter.
 
+# **Q: How to create a batch of DIDs?
+_Can you explain more detail on this? Assume that the company requires to create 1000 DIDs for IOT devices. How can I do create them in a batch?_
+
+You'd use the API to send your running node N number of ops, then it will batch them up for you and anchor them (assuming you attach a funded wallet that can do txns).\
+Assuming you are running your own node, you simply need to submit 1000 create operations to your node (ie. http://localhost:3000/operations) before your batch writer kicks in every 10 minutes by default, the batch writer will batch all 1000 operations into 1 bitcoin transaction thus you'll pay fee for just one transaction to the minor.\
+The technical spec for constructing a create request can be found in [Sidetree API spec](https://identity.foundation/sidetree/api/#create), but if you know TypeScript, you are better off using the [ion-sdk](https://github.com/decentralized-identity/ion-sdk) directly, it will save you a lot of time.\
+(_@TheHenryTsai_) - related github [issue](https://github.com/decentralized-identity/ion/issues/210)
+
+#### *Q: Does it cost only a single bitcoin transaction or 1000 BTC transactions?
+To create 1000 DIDs for IOT devices, you wouldn't need 1000 BTC transactions, but only a fraction. Furthermore costs can be shared among many users. e.g. ION is able to bundle thousands of pluriform identifier events in a Merkletree-like structure and only the root hash will be anchored on the bitcoin blockchain. It depends whether you're able to batch the inception events of the IoT identifiers or not. A batch of 1000 identifiers requires 1 BTC transaction and there's even more room in that ION batch.\
```

Comment:
Misconception, there is actually no merkle tree used for batch. So replace:

`in a Merkletree-like structure and only the root` to `and only a`